change/@fluentui-react-drawer-54014221-befb-409e-a4a3-543c1570d111.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31668&commitId=b795c2dcc554c1844796842ddf92032068f60cd2&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|6| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|7| Changed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Drawer</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.default drawer - High Contrast.chromium.png|1529| Changed|
|Drawer.default drawer - Dark Mode.chromium.png|1529| Changed|
|Drawer.default drawer - RTL.chromium.png|272| Changed|
|Drawer.default drawer.chromium.png|272| Changed|
|Drawer.overlay drawer - High Contrast.chromium.png|1529| Changed|
|Drawer.overlay drawer - Dark Mode.chromium.png|1529| Changed|
|Drawer.overlay drawer alert.chromium.png|272| Changed|
|Drawer.overlay drawer alert - High Contrast.chromium.png|1529| Changed|
|Drawer.overlay drawer large content scroll - Dark Mode.chromium.png|1529| Changed|
|Drawer.overlay drawer - RTL.chromium.png|272| Changed|
|Drawer.overlay drawer alert - Dark Mode.chromium.png|1529| Changed|
|Drawer.overlay drawer alert - RTL.chromium.png|272| Changed|
|Drawer.overlay drawer large content scroll - High Contrast.chromium.png|1529| Changed|
|Drawer.overlay drawer large - High Contrast.chromium.png|1529| Changed|
|Drawer.overlay drawer large - Dark Mode.chromium.png|1529| Changed|
|Drawer.overlay drawer large - RTL.chromium.png|272| Changed|
|Drawer.overlay drawer medium - Dark Mode.chromium.png|1529| Changed|
|Drawer.overlay drawer large.chromium.png|272| Changed|
|Drawer.overlay drawer large content scroll - RTL.chromium.png|272| Changed|
|Drawer.overlay drawer large content scroll.chromium.png|272| Changed|
|Drawer.overlay drawer medium - High Contrast.chromium.png|1529| Changed|
|Drawer.overlay drawer non modal - High Contrast.chromium.png|2996| Changed|
|Drawer.overlay drawer medium - RTL.chromium.png|272| Changed|
|Drawer.overlay drawer non modal - RTL.chromium.png|995| Changed|
|Drawer.overlay drawer medium.chromium.png|272| Changed|
|Drawer.overlay drawer non modal.chromium.png|995| Changed|
|Drawer.overlay drawer position end - High Contrast.chromium.png|1529| Changed|
|Drawer.overlay drawer position end - Dark Mode.chromium.png|1529| Changed|
|Drawer.overlay drawer position end - RTL.chromium.png|272| Changed|
|Drawer.overlay drawer non modal - Dark Mode.chromium.png|2996| Changed|
|Drawer.overlay drawer position start - Dark Mode.chromium.png|1529| Changed|
|Drawer.overlay drawer position end.chromium.png|272| Changed|
|Drawer.overlay drawer position start - High Contrast.chromium.png|1529| Changed|
|Drawer.overlay drawer.chromium.png|272| Changed|
|Drawer.overlay drawer position start - RTL.chromium.png|272| Changed|
|Drawer.overlay drawer position start.chromium.png|272| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Removed|
|SwatchPicker Converged.default - Dark Mode.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>